package/yast2-nfs-client.spec

```diff
@@ -41,6 +41,12 @@ Requires:       yast2-nfs-common >= 2.24.0
 # showmount, #150382, #286300
 Recommends:     nfs-client
 
+# Unfortunately we cannot move this to macros.yast...
```

Comment:
It is not obvious why. Please reference a bug / issue that will say "bcond within macros are ignored by  prepare_build" or something like that.